runway/cfngin/hooks/aws_lambda.py

```diff
@@ -464,6 +464,29 @@ def dockerized_pip(work_dir, client=None, runtime=None, docker_file=None,
         LOGGER.info('lambda.docker: %s', log.decode().strip())
 
 
+def _pip_has_no_color_option(python_path):
```

Comment:
This probably does not need to be a protected function and it should have test written for it.

Reply:
I was torn on whether to make it open. I decided not to because I think we're likely to toss it entirely in the relatively near future, and its so dedicated to this purpose that I don't want any external use of the function to impede us getting rid of it.

I'll see about testing it

Reply:
>  I think we're likely to toss it entirely in the relatively near future

if this is the case, we should have a timeline for removal added noted somewhere.

honestly, not sure who could put up with >2 years of pip telling you to update it each time something is installed to still be using <v10.

Reply:
> honestly, not sure who could put up with >2 years of pip telling you to update it each time something is installed to still be using <v10.
this.jpg